app/src/main/java/com/practicum/appplaylistmaker/AudioplayerActivity.kt

```diff
@@ -13,21 +18,69 @@ import com.google.gson.Gson
 import com.practicum.appplaylistmaker.api.Track
 
 class AudioplayerActivity : AppCompatActivity() {
+
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+    }
+
+    private lateinit var musicTimer: TextView
+    private var playerState = STATE_DEFAULT
+    private lateinit var buttonPlayStop: ImageButton
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Здесь лучше имьютабельной сделать переменную, т.к. плеер не требуется пересоздавать, лучше переиспользовать один объект. Создание такого плеера - затратная операция